CMSgov/easi-app
# [Easi-1902] Create "Prepare for the Governance Review Board meeting" help article

# EASI-1902

## Changes and Description

- Add new `PrepareforGRB` component in the help sections
- Add new routes for `PrepareforGRB`
- Add new copy to the i18n.ts files




## How to test this change

1. Navigate to http://localhost:3000/help
2. Find and click on article titled: `Prepare for the Review Board meeting`
3. Ensure new tab opens with the article.
4. Ensure article look and copy reflects figma
 
## JIRA Acceptance Criteria

- Reuse existing 'Prepare for GRB Meeting' view/component
- May need to tweak existing component to be reusable in any area within EASI
- Testing where applicable
